.github/workflows/bash_code_analysis.yaml

```
@@ -36,7 +36,7 @@ jobs:
       - name: Install shfmt
         uses: mfinelli/setup-shfmt@v4
       - name: shfmt scan
-        run: shfmt -d dev/jobs jobs scripts
+        run: shfmt -d dev/jobs dev/job_cards scripts
```

I think `scripts/` are now in `dev/scripts`
```suggestion
        run: shfmt -d dev/jobs dev/job_cards dev/scripts
```